.github/workflows/sync.yml

```diff
@@ -0,0 +1,40 @@
+name: Upstream Sync
```

Comment:
This file is only for WTFAcademy. Could you change your base branch on the AmazingAng:main. Not merge the WTFAcademy fork's repo.

Reply:
> This file is only for WTFAcademy. Could you change your base branch on the AmazingAng:main. Not merge the WTFAcademy fork's repo.

do you mean to change my base branch from Mosamorphing/WTF-Solidity:szn to Mosamorphing/WTF-Solidity:main before merge?

Reply:
<img width="651" alt="image" src="https://github.com/AmazingAng/WTF-Solidity/assets/72180730/da891f7b-b39e-4d01-9b62-3265753e44f2">

These commits do not belong to AmazingAng/WTF-Solidity repo, there are only for WTFAcademy/WTF-Solidity repo.

Reply:
> > This file is only for WTFAcademy. Could you change your base branch on the AmazingAng:main. Not merge the WTFAcademy fork's repo.
> 
> do you mean to change my base branch from Mosamorphing/WTF-Solidity:szn to Mosamorphing/WTF-Solidity:main before merge?

You merge some commits that do not belong to this repo.

Reply:
> > > This file is only for WTFAcademy. Could you change your base branch on the AmazingAng:main. Not merge the WTFAcademy fork's repo.
> > 
> > 
> > do you mean to change my base branch from Mosamorphing/WTF-Solidity:szn to Mosamorphing/WTF-Solidity:main before merge?
> 
> You merge some commits that do not belong to this repo.

oh, I see. i think the issue may be arising from creating the PR from the WTFAcademy/WTF-Solidity repo as I was not able to originally fork the AmazingAng/WTF-Solidity repo. is there a way to manually edit out this commit and bypass the fork showcasing the main repo? if yes, kindly show how.

Reply:
Sorry, I was too busy and didn't see your reply. You can cherry-pick commits, except for the one that 'Create Sync.yaml'.

Reply:
> > > > This file is only for WTFAcademy. Could you change your base branch on the AmazingAng:main. Not merge the WTFAcademy fork's repo.
> 
> > > 
> 
> > > 
> 
> > > do you mean to change my base branch from Mosamorphing/WTF-Solidity:szn to Mosamorphing/WTF-Solidity:main before merge?
> 
> > 
> 
> > You merge some commits that do not belong to this repo.
> 
> 
> 
> oh, I see. i think the issue may be arising from creating the PR from the WTFAcademy/WTF-Solidity repo as I was not able to originally fork the AmazingAng/WTF-Solidity repo. is there a way to manually edit out this commit and bypass the fork showcasing the main repo? if yes, kindly show how. 

Sorry, I was too busy and didn't see your reply. You can cherry-pick commits, except for the one that 'Create Sync.yaml'.

Reply:
And we will squash your commits to one commit.

Reply:
i don't want to mess this up any further, can we hash this out over a short call on discord? @XdpCs